### PR TITLE
CD-XXX-Product UI fixes

### PR DESCRIPTION
**Pull request by Chelsea-davids6**
- Fixed auto refresh on product screen
- Fixed cancel button on product screen
- Fixed notification validation on notification screen

### CODE REVIEW

File: src/pages/notifications.vue [hunk 1]
```diff
@@ -261,36 +261,41 @@
...
-         } 
-         finally {
-            loading.value = false;
-            newDialog.value = false;
-            saved.value = false
-         }
+      saved.value = true;
+
+   if (!newNotification.value.message) {
+      console.error("Notification message is empty. Please enter a message.");
```
Comment (AdamKruger1212) [inline src/pages/notifications.vue]: rather remove this. we already have an error on the screen and its bad practice to log ui erros or ui related errors in the console on production

File: src/pages/notifications.vue [hunk 2]
```diff
@@ -261,36 +261,41 @@
...
+      return; 
+   }
+
+   console.log(selectedProducts.value);
```
Comment (AdamKruger1212) [inline src/pages/notifications.vue]: remove console.log, never display user data in console

File: src/pages/notifications.vue [hunk 3]
```diff
@@ -261,36 +261,41 @@
...
+   var payload;
+   const formattedUserIds = selectedUsers.value.map(item => ({ "user_id": item.sid.toString() }));
+   const formattedPolicyIds = selectedProducts.value.map(item => ({ "policy_id": item.sid.toString() }));
+   console.log(formattedUserIds);
```
Comment (AdamKruger1212) [inline src/pages/notifications.vue]: remove console

**Review by AdamKruger1212 — CHANGES_REQUESTED** [review]

File: src/pages/notifications.vue (hunk 2 above)
Comment (Chelsea-davids6) [reply src/pages/notifications.vue]: i have fixed it

File: src/pages/notifications.vue (hunk 3 above)
Comment (Chelsea-davids6) [reply src/pages/notifications.vue]: This has been done

File: src/pages/notifications.vue (hunk 1 above)
Comment (Chelsea-davids6) [reply src/pages/notifications.vue]: I fixed it

**Review by AdamKruger1212 — APPROVED** [review]

**Pull request merged**